modules/ROOT/pages/proc_deploying-an-application-to-openshift.adoc

```diff
@@ -36,6 +36,6 @@ $ oc get serviceaccount pipeline
 
 . In the OpenShift Web Console switch to the *Developer* perspective of the OpenShift Web Console.
 
-. Select the `pipelines-tutorial` project from the *Project* drop-down menu. On the *Topology* view of the *Developer* perspective, you will be able to see the resources of your Pipeline. The deployment at this stage is not complete. The `Pipeline` you create in the next steps builds the simple application which has a frontend and a backend and completes the deployment.
+. Select the `pipelines-tutorial` project from the *Project* drop-down menu. On the *Topology* view of the *Developer* perspective, you will be able to see the resources of your Pipeline. The deployment at this stage is not complete. The `Pipeline` you create in the next steps builds the simple application which has a *frontend* and a *backend* and completes the deployment.
```

Comment:
`pipelines-tutorial`>*pipelines-tutorial*
Remove bold format for frontend and backend